arangod/Aql/RegisterPlan.h

```diff
@@ -151,7 +151,6 @@ struct RegisterPlanT final : public std::enable_shared_from_this<RegisterPlanT<T
 
   auto variableToRegisterId(Variable const* variable) const -> RegisterId;
 
-  // compatibility function for 3.6. can be removed in 3.8
```

Comment:
Is the comment inaccurate, or should the function be removed?

Reply:
Unfortunately, the comment was inaccurate :smile: